packages/expo-updates/expo-updates-gradle-plugin/src/main/kotlin/expo/modules/updates/ExpoUpdatesPlugin.kt

```diff
@@ -126,3 +127,22 @@ private fun detectedEntryFile(config: ReactExtension): File {
     else -> File(reactRoot, "index.js")
   }
 }
+
+private fun isNativeDebuggingEnabled(config: ReactExtension): Boolean {
+  if (System.getenv("EX_UPDATES_NATIVE_DEBUG") == "1") {
+    return true
+  }
+  val properties = Properties()
+  val reactRoot = config.root.get().asFile
+  val androidRoot = File(reactRoot, "android")
+  val gradlePropertiesFile = File(androidRoot, "gradle.properties")
+  when (gradlePropertiesFile.exists()) {
```

Comment:
i think you can still use [`project.findProperty`](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api/-project/find-property.html) here. that would save direct file parsing for gradle.properties

Reply:
Thanks very much! I simplified the method to use `project.findProperty()`